### ISSUE TITLE
Possible to Sponsor the Community Edition?

### ISSUE THREAD

**Issue by inmanturbo** [skeema/skeema#172]
I understand that there is a premium edition to this software, but I would like to sponsor the community edition. Would that be possible?

I am only using it for labs and development ATM but I do have my eye on the Premium for future projects with production deployments. In the meantime I would like to be able to do what I can out of pocket to help with the CE.

**Comment by evanelias** [comment]
Thank you for offering, but individual sponsorships are not possible at this time. Community edition feature development is largely on hold right now, and accepting sponsorships could lead to unrealistic expectations from users.

Although Skeema is used by several hundred companies and installed over 30,000 times per month, only a tiny percentage are subscribing to the premium products so far. As a result, all feature development energy is strictly going towards the premium products right now. These products need to become more appealing to a larger percentage of the user base, so that Skeema will have a sustainable revenue stream from corporate customers. Without this, it will not be possible for Community Edition development to continue.

In other words, subscribing to the premium products is the way to support Skeema. In addition to the [Premium CLI](https://www.skeema.io/download/) (starts at $99/month for the Plus tier), there's also the separate development/CI-focused SaaS product [Skeema Cloud Linter](https://www.skeema.io/download/#skeema-cloud-linter-for-github) which is only $59/month per GitHub organization. Thank you for your future consideration.

**Comment by inmanturbo** [comment]
>  accepting sponsorships could lead to unrealistic expectations from users.

@evanelias
 
Expectations or no, Skeema is under no obligation to satisfy such expectations on a community driven open source product. There will always be members of any community with unrealistic expectations -- it cannot be helped. 

I only wish to sponsor because I understand the difficulty in _maintaining_ such community projects, not because I expect additional features. _I would hate to see the project die_, or become archived and unusable without important bug fixes and updates before I even get a chance to finish testing/tinkering with it and convince a client to let us use it professionally.

So if revenue is an issue, why turn down free money? It is not feasible for me to pay Premiums on tools that I am not using in any way that generates revenue on my end, however myself and many others I'm sure would be more than willing to fork over some pocket money to support continued maintenance something we find exceptional, or even just enjoy tinkering with.

Given the amount of interest surrounding skeema/skeema it has the potential to inspire a good deal of sponsorship. Many projects on GH bring in tens and even hundreds of thousands of dollars in sponsorship.

I understand it may not be possible right now, but please give me an @ if that ever changes.

Thank you for sharing your considerable feats in engineering. I wish that I could do more to support the project.

**Comment by evanelias** [comment]
First, let me say I understand you have good intentions with your post. But frankly I find this discussion extremely frustrating and draining.

I'm trying to run a business here. The premium products are very competitively priced relative to the other two main commercial tools in this space. Despite this, you seem to be signaling that this price does not make sense for your use-case at this time. This is perfectly understandable. But you're also saying you'd be happy to pay "pocket money" instead in order to support ongoing maintenance; and furthermore, you are signaling that you expect *many other users* would be interested in this model, despite them not being interested in the premium products.

Indirectly, this would seem to mean that the premium products are overpriced, and/or not compelling in functionality. Maybe this is the case, but if so the solution would be to *focus more time on improving the premium products*, which is precisely what is already happening. 

Database automation is time-consuming, and generally it pays well. My time is valuable, and I don't work for pocket change. I am not interested in developing or maintaining database automation software under a pay-what-you-wish sponsorship model, nor is it feasible for me to reduce the prices of the premium products below their already-modest levels. I also do not believe a sponsorship model would amount to a meaningful revenue stream; from what I've seen the *vast* majority of GitHub Sponsors projects take in *massively* less than you are suggesting here. I'm familiar with several projects that are much more popular than Skeema, but take in under $1k/month. That simply isn't enough revenue to make any impact whatsoever on the continued viability of the Community edition.

I personally haven't derived much of any benefit from the existence of the Community edition to date, and it does not make sense for me to continue pouring my time into it indefinitely. I've already put the equivalent of 2.5 years of unpaid full-time work into the Community edition. Enough is enough. If it doesn't start driving a meaningful conversion rate to the premium products, or driving consulting business, or driving support contracts, then it's a nonviable product and I really can't justify spending any more time on it.

If you would like to support Skeema but the premium products aren't a fit at this time, "spreading the word" is always appreciated -- blogging, tweeting, word-of-mouth, etc.

**Comment by inmanturbo** [comment]
@evanelias 
I seem to have struck a nerve. Perhaps I should have chosen my wording more carefully, or simply let you be. It is your project after all. 

It is certainly viable. We just just don't start many new projects (we just do a few big ones) and I wasn't aware of it at the start of the last, and am not in a position to be able to replace everything already in place.

I've only recently became aware of the project and have been tinkering around with it and have been very impressed.

It's not overpriced nor am I meaning to suggest it is. It's just soon. And I don't blog and I don't tweet and I don't intend for donations to take time away from working on the premium products. I don't handle marketing. I do handle purchasing.

By "pocket money" I was referring more to the source than the quantity. You may surprised to find that I have much more budgetary freedom when it comes to sponsoring open source projects than purchasing licenses. Not that it matters at this point. I wish you all the best, and good luck in your endeavors.
